apps/root/src/frame/components/navigation/index.tsx

```diff
@@ -83,25 +83,25 @@ const Navigation = ({
   React.useEffect(() => {
     if (HOME_ROUTES.includes(location.pathname)) {
       dispatch(changeRoute('home'));
-      pushToHistory('/');
+      // pushToHistory('/');
```

Comment:
Should remove this comments, may be confusing in the future 😄

Reply:
Oh thanks